docs/config/README.md

```diff
@@ -1722,6 +1722,8 @@ format = 'on [⛵ $context \($namespace\)](dimmed green) '
 disabled = false
 [kubernetes.context_aliases]
 "dev.local.cluster.k8s" = "dev"
+".*/openshift-cluster/.*" = "openshift"
```

Comment:
Can you expand the documentation a bit more. This is a really cool feature and it would be better to make this easier to find.

Reply:
Yeah of course, what do you have in mind, an extra section explaining the regex matching?

Reply:
> Yeah of course, what do you have in mind, an extra section explaining the regex matching?

That would be perfect 👍🏻

Reply:
Added an additional section, please let me know if I can improve or if there are any grammar mistakes.